TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift

```diff
@@ -179,7 +179,9 @@ public class TwilioVerifyBuilder {
     return self
   }
   
-  func setClearStorageOnReinstall(_ clearStorageOnReinstall: Bool) -> Self {
+  /// Defines if the storage will be cleared after a reinstall
+  /// - Parameter clearStorageOnReinstall: If true, the storage will be cleared after a reinstall, so created factors will not exist in the device anymore. If false, created factors will persist in the device. Default value is true
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/6cb8fbf27f8bc164af4278ce9c57e362b1354c81/TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift#L183"></span>Line should be 195 characters or less: currently 231 characters<br />
<code>line_length</code> <code>TwilioVerify.swift:183</code></td>
    </tr>
  </tbody>
</table>